roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -219,6 +219,20 @@ driver = log
 [oslo_middleware]
 enable_proxy_headers_parsing = True
 
+[placement]
+auth_url = {{ endpoints.keystone.url.admin }}/{{ endpoints.keystonev3.version }}
+auth_type = password
+username = placement
+password = {{ secrets.service_password }}
+cafile = {{ nova.cafile }}
+project_name = service
+project_domain_id = default
+os_region_name = RegionOne
+user_domain_id = default
+{% if insecure|default('False')|bool %}
+insecure = true
+{% endif %}
+
```

Comment:
I read the upstream code and docs and this is not needed. I am still validating without it.